index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="style/main.css">
+    <title>Apple 2014</title>
+</head>
+<body>
+    <header class="flex col">
+        <nav id="main-nav" class="flex">
+            <div class="nav-item nav-link flex" id="brand-icon-div">
+                <i class="fab fa-apple"></i>
+                <div class="overlay flex">
+                    <i class="fab fa-apple"></i>
+                </div>
+            </div>
+            <div class="nav-item nav-link flex">Store
+                <div class="overlay flex">Store</div>
+            </div>
+            <div class="nav-item nav-link flex">Mac
+                <div class="overlay flex">Mac</div>
+            </div>
+            <div class="nav-item nav-link flex">iPod
+                <div class="overlay flex">iPod</div>
+            </div>
+            <div class="nav-item nav-link flex">iPhone
+                <div class="overlay flex">iPhone</div>
+            </div>
+            <div class="nav-item nav-link flex">iPad
+                <div class="overlay flex">iPad</div>
+            </div>
+            <div class="nav-item nav-link flex">iTunes
+                <div class="overlay flex">iTunes</div>
+            </div>
+            <div class="nav-item nav-link flex">Support
+                <div class="overlay flex">Support</div>
+            </div>
+            <div class="nav-item flex" id="search-div">
+                <span><i class="fas fa-search"></i></span>
+                <input type="text" id="search">
```

Comment:
What about making the background white on `:hover` ?

Reply:
We did try to do that, on focus but  because of how we positioned the icon in the search (absolutely - relative to the search div) using `input:hover` or `input:focus` to change color is a bit difficult because We'd have to use  `input:focus + .fa-search ` etc. to change color of icon as well to grey but it does not work because it is out of normal flow. Do you have nay suggestions?